### PR TITLE
Add better currency formating

### PR DESCRIPTION
**Pull request by incik**

### CODE REVIEW

File: src/format.ts [hunk 1]
```diff
@@ -55,13 +58,21 @@ export function formatNumber(input: number, givenOptions: FormatOptions) {
...
   const options = mergeOptionsWithDefault(givenOptions);
-  const numbers = input.toString().split('.');
+  const numbers = input.toFixed(givenOptions.precision).toString().split('.');
 
   numbers[0] = numbers[0].replace(/\B(?=(\d{3})+(?!\d))/g, options.delimiter);
 
-  return numbers.join(options.separator);
+  const ret = numbers.join(options.separator);
+  if (givenOptions.trimTrailingZeros) {
+    return ret.replace(/0+$/g, '');
```
Comment (honzabrecka) [inline src/format.ts]: Shouldn't be this done on `numbers[1]`? `3400` is trimmed like this `34`. Not sure about it.

Comment (incik) [reply src/format.ts]: Damn, thank you! I'll write test for it and try to fix it :)

Comment (incik) [reply src/format.ts]: I've found different problem :) Now there is trailing dot after whole numbers :-D

Comment (incik) [reply src/format.ts]: I made small refactoring of the removing of trailing zero. Should be OK now.

**Review by honzabrecka — APPROVED** [review]

**Review by ondrejbartas — APPROVED** [review]

**Pull request merged**